detect_secrets/core/audit.py

```diff
@@ -345,11 +346,16 @@ def get_secrets_list_from_file(baseline_filename: str) -> list:
 
 def _get_baseline_from_file(filename):  # pragma: no cover
     try:
+        if not os.path.exists(filename):
+            raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), filename)
         with open(filename) as f:
             return json.loads(f.read())
+    except (FileNotFoundError):
+        print('File not found: {}'.format(filename), file=sys.stderr)
+        sys.exit(errno.ENOENT)
```

Comment:
https://docs.python.org/3/library/errno.html#errno.ENOENT

Reply:
Out of curiosity do we not use a standard logger for this? Because if so, we could just do `logger.error` instead